src/python/turicreate/toolkits/_model.py

```diff
@@ -640,6 +641,9 @@ def name(self):
         """
         Returns the name of the model.
 
+        ..WARNING:: This function is deprecated, It will be removed in the next
+        major release. Use type(CustomModel) instead.
```

Comment:
Sounds like we're going to remove this in 6.2. So that's not "the next major release". It's just the next release.

Reply:
Good!